src/tailwind/plugins/base/gc-base.js

```diff
@@ -9,6 +9,19 @@ module.exports = function () {
         scrollBehavior: 'smooth',
         overflowY: 'visible !important', // Need this for sticky nav to work
       },
+      body: {
```

Comment:
Tell me more about this change...

Reply:
Had double long meetings with Kerri and Ali today. This is the new base font size that Kerri feels like makes the most sense after working on a few modern projects. All the other typography styles are in em unit so will just scale to the base at all breakpoints 😄